extensions/random.py

```
@@ -19,7 +19,7 @@
 # RANDOM -> CLASS
 class _random:
     # CLASS -> INIT
-    async def init(self, request: Request) -> namespace: return self
+    async def init(self, request: Request, response: Response) -> namespace: return self
```

Ruff (0.12.2)

`Request` may be undefined, or defined from star imports

(F405)